cmd/ddev/cmd/exec.go

```diff
@@ -45,15 +45,13 @@ ddev exec --raw -- ls -lR`,
 		opts := &ddevapp.ExecOpts{
 			Service: serviceType,
 			Dir:     execDirArg,
-			Cmd:     strings.Join(args, " "),
+			Cmd:     quoteArgs(args),
```

Comment:
I'm not convinced that it's appropriate to do this when using ExecRaw. Definitely want to talk about it.

Reply:
`Cmd` is not used when `RawCmd` is provided, I will set `opts.Cmd = ""` for `--raw` explicitly to make it clear.

Reply:
I got it, the problem was that I modified `args` inside `quoteArgs(args)`
I thought there was a copy inside, but it was a reference.